metal/v1/api_console_log_details.go

```diff
@@ -0,0 +1,175 @@
+
+API version: 1.0.0
+Contact: support@equinixmetal.com
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
+package v1
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+)
+
+// ConsoleLogDetailsApiService ConsoleLogDetailsApi service
+type ConsoleLogDetailsApiService service
+
+type ApiCaptureScreenshotRequest struct {
+	ctx        context.Context
+	ApiService *ConsoleLogDetailsApiService
+	id         string
+}
+
+func (r ApiCaptureScreenshotRequest) Execute() (*os.File, *http.Response, error) {
+	return r.ApiService.CaptureScreenshotExecute(r)
+}
+
+/*
+CaptureScreenshot Method for CaptureScreenshot
+
+Capture a screenshot from the device, if supported, via the BMC.
+
+	@param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+	@param id Device UUID
+	@return ApiCaptureScreenshotRequest
+*/
+func (a *ConsoleLogDetailsApiService) CaptureScreenshot(ctx context.Context, id string) ApiCaptureScreenshotRequest {
```

Comment:
Added the screenshot API, but I'm pretty sure access to that is limited to specific customers